Functions.Templates/Templates-v2/TimerTrigger-Python/template.json

```diff
@@ -57,7 +57,7 @@
                     "type": "AppendToFile",
```

Comment:
Actions within a job should just be a list of strings right? This has a json. Shouldn't this json be in the actions section below?

Reply:
This is an example of an anonymous action. There should be a section explaining when and how you would define one in the internal walkthrough doc (which I should pull over here) but it's an alternative way of defining actions specific to a single job.